src/precice/impl/SolverInterfaceImpl.cpp

```diff
@@ -1473,46 +1473,52 @@ void SolverInterfaceImpl::computePartitions()
   }
 }
 
-void SolverInterfaceImpl::mapWrittenData()
+void SolverInterfaceImpl::computeMappings(utils::ptr_vector<MappingContext> contexts, std::string mappingType)
 {
-  PRECICE_TRACE();
```

Comment:
Why do you delete the trace here?

Reply:
Added `PRECICE_TRACE` to all the new functions in 9ef9db27efbadeccb2fdf3670e54d667f5846f6f. I did not intentionally delete the trace here. I rather forgot to add it in the new functions.

Reply:
I must admit I don't know where it should ideally go and where not. No we have it in both: the callee and the caller function. Feel free to merge if you know are familiar :)

Reply:
@DavidSCN: If I get the tracer right it can go at the beginning of each function to tell the logger which function is entered. See here:

https://github.com/precice/precice/blob/bd0e158aa0cc795ba0c7b227cf1c086c73cbc896/src/logging/LogMacros.hpp#L42-L45